src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -724,3 +724,54 @@ def test_wait_for_event_repeated_capabilities(session, channels):
     channels_session = session.channels[channels]
     with channels_session.initiate():
         channels_session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE)
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+@pytest.mark.parametrize(
+    "compensation_function",
+    [
+        nidcpower.Session.perform_lcr_open_custom_cable_compensation,
+        nidcpower.Session.perform_lcr_short_custom_cable_compensation,
+    ],
+)
+def test_perform_lcr_open_short_custom_cable_compensation(session, compensation_function):
+    compensation_function(session)
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+def test_lcr_custom_cable_compensation_data(session):
+    compensation_data = session.get_lcr_custom_cable_compensation_data()
+    session.configure_lcr_custom_cable_compensation(compensation_data)
```

Comment:
after every configure using different types, can you "get" and assert it is the expected values?
otherwise we can't tell if things got mangled

Reply:
The current driver (NI-DCPower 21.8) doesn't really support "getting" the "configured" LCR custom cable compensation data, as `get_lcr_custom_cable_compensation_data()` will always retrieve the custom cable compensation data from the onboard storage (i.e. previously generated data) instead of the latest data configured through `configure_lcr_custom_cable_compensation()`.
![image](https://user-images.githubusercontent.com/80089084/169737996-aec9fe22-91a4-406a-a3f5-02e485d1846d.png)